### PR TITLE
Translate 2015-12-11-ruby-2-3-0-preview2-released (ja)

### PR DESCRIPTION
**Pull request by makimoto**
Translation of [2015-12-11-ruby-2-3-0-preview2-released](https://www.ruby-lang.org/en/news/2015/12/11/ruby-2-3-0-preview2-released/) into Japanese.

@ruby/www-ruby-lang-org-i18n-ja Please review it.

### CODE REVIEW

File: ja/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+lang: ja
+---
+
+Ruby 2.3.0-preview2 がリリースされました。
+
+Ruby 2.3.0-preview2 は Ruby 2.3.0 の2つ目のプレビューです。
+多くの新機能と改善が含まれます。
+
+Ruby 2.3.0-preview2 is the second preview of Ruby 2.3.0.
+Many new features and improvements are included.
```
Comment (marocchino) [inline ja/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md]: need to delete.

File: ja/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+[Frozen String Literal Pragma](https://bugs.ruby-lang.org/issues/11473) が導入されました。
+Ruby 2.1 から `"str".freeze` はオブジェクトアロケーションを削減するように最適化されています。
+Ruby 2.3 はソースファイル内の滑ての文字列リテラルを不変にする新しいマジックコメントとコマンドラインオプションが導入されます。
+
+またデバッグ用に、`--debug=frozen-string-literal` コマンドラインオプションによって、
+`"can't modify frozen String"` エラーが発生したときにそのオブジェクトがどこで生成されたかを知ることができます。
+
+[safe navigation operator](https://bugs.ruby-lang.org/issues/11537) (またの名を[ぼっちオペレータ](https://instagram.com/p/-M9l6mRPLR/)) `&.` は、既に C#、Groovy、Swift で存在しているもので、
```
Comment (marocchino) [inline ja/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md]: The safe navigation operator is exists in "C#、Groovy、Swift" but, `&.` is not.
How about this: 
> [safe navigation operator](https://bugs.ruby-lang.org/issues/11537)は、既に C#、Groovy、Swift で存在しているもので、(またの名を[ぼっちオペレータ](https://instagram.com/p/-M9l6mRPLR/) `&.`)

Comment (makimoto) [inline ja/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md]: I split it into 2 sentences as 9113712ee323938f436789e7bc9534514e2006d5.
How about it?

Comment (marocchino) [inline ja/news/_posts/2015-12-11-ruby-2-3-0-preview2-released.md]: nice!

**Pull request merged**